tests/conftest.py

```diff
@@ -16,3 +25,64 @@ def setup_env(pytestconfig, request):
 
     if not running_unit_tests and environment in ['local', 'qa']:
         load_env_file(environment, file_string=f'config/{environment}.yaml')
+
+@pytest.fixture(scope='module')
+def db_manager():
+    db_manager = DBManager()
+    db_manager.createSession()
+    yield db_manager
+    db_manager.close_connection()
```

Comment:
hell ya - looks super clean. nice way to do this to make sure we are closing the connection :)